doc/developer/debugging.rst

```diff
@@ -16,9 +16,13 @@ Using a browser-integrated debugging tool such as
 or Chrome development tool is very useful.
 
 In case of 500 errors have a look at the apache logs, located at 
-``/var/www/vhosts/<vhost_name>/logs/error.log``. If we call a service behind
+``/var/www/vhosts/<vhost_name>/logs/<vhost_name>_error.log``. If we call a service behind
 a proxy, the log entry actually references the final URL. Therefore you may
-call the latter URL directly on the server by typing ``curl "<url>"``.
+call the latter URL directly on the server by typing
+``curl "http://localhost/<path>" -H Host:<server_name>``.
+
+It the ``pyramid_debugtoolbar`` is enable the error is directly return in the
```

Comment:
It => If
enable => enabled
return => returned
query in error => query that fails